### PR TITLE
Sorted movies

### PR DESCRIPTION
**Pull request by akshita27**

### CODE REVIEW

File: imdb/sort_movies.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+echo "" > imdbResponse.txt
+for file in $1/*
+do
+        enc_file=$(echo ${file##*/} | sed "s/ /%20/g" | sed "s/'/%27/g")
+	printf '%s' "$enc_file " >> imdbResponse.txt
+	curl -s  http://www.omdbapi.com/\?i\=\&t\=$enc_file | jq '.imdbRating'| sed -e 's/^"//'  -e 's/"$//' >> imdbResponse.txt
```
Comment (vishesh92) [inline imdb/sort_movies.sh]: https://google.github.io/styleguide/shell.xml#Line_Length_and_Long_Strings

Comment (akshita27) [inline imdb/sort_movies.sh]: will just work on it and improve the styling and indentation

File: imdb/sort_movies.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+echo "" > imdbResponse.txt
+for file in $1/*
+do
+        enc_file=$(echo ${file##*/} | sed "s/ /%20/g" | sed "s/'/%27/g")
```
Comment (vishesh92) [inline imdb/sort_movies.sh]: https://google.github.io/styleguide/shell.xml#Indentation

File: imdb/sort_movies.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+echo "" > imdbResponse.txt
```
Comment (vishesh92) [inline imdb/sort_movies.sh]: shebang?

**Pull request merged**